jipdate.py

```diff
@@ -498,6 +498,20 @@ def get_header():
 
     return ("\n".join(yml_iter) + "\n\n") if yml_iter is not None else "\n"
 
+def get_editor():
+    global g_yml_config
+
+    try:
+        yml_iter = g_yml_config['text-editor']
+    except:
+        # Probably no "text-editor" section in the yml-file.
+        return True
+
+    if yml_iter == "no" or yml_iter == "NO" or yml_iter == "No":
```

Comment:
For the code flow? No. But I think it's a nice and easy way to create a config.yml that contains the various settings. As it is right now, you need to know the script to know what you can put in config.yml. If we default populate that, then it's a no-brainer for anyone open the file to figure out what to do.